provision/dev/minio_create_apikey.sh

```diff
@@ -7,9 +7,10 @@ MINIO_USER=${1:?MINIO_USER}
 MINIO_PASS=${2:?MINIO_PASS}
 ACCESS_KEY=${3:?ACCESS_KEY}
 SECRET_KEY=${4:?SECRET_KEY}
+KUBECONFIG_CONTEXT=${5}
```

Comment:
`:?<TEXT>` is useful for debugging undefined variables.

```suggestion
KUBECONFIG_CONTEXT=${5:?KUBECONFIG_CONTEXT}
```

Reply:
The issue is that `KUBECONFIG_CONTEXT` is set to an empty value for the single cluster case.

Reply:
Ah, I see. LG!

Reply:
Thanks! Merging.